frontend/src/components/Routes/Presentational.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { UserState } from 'redux-oidc';
+import { IRegisteredUserState } from '../RegisterUser/reducer';
+import { RoutesLogin } from '../RoutesLogin';
+import { RoutesRegister } from '../RoutesRegister';
+import { RoutesUser } from '../RoutesUser';
+
+interface IDispatchProps {
+  fetchUserData: () => void;
+}
+
+interface IUserStates {
+  userInformation: IRegisteredUserState;
+  oidc: UserState;
+}
+
+type Props = IUserStates & IDispatchProps;
+
+export class Presentational extends React.Component<Props, {}> {
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public render() {
+    const { oidc, userInformation } = this.props;
+    if (!oidc.user || oidc.user.expired) {
```

Comment:
Can be rewritten 
 if (!oidc.user || oidc.user.expired) return <RoutesLogin />;

if you want